src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/auth/LogoutUseCase.java

```diff
@@ -0,0 +1,10 @@
+package de.dhbw.karlsruhe.students.mailflow.core.application.auth;
+
+
+public interface LogoutUseCase {
+  /**
+   * @return the address of the user that was logged out
+   * @throws IllegalStateException when there is no logged-in user
+   */
+  String logout() throws IllegalStateException;
```

Comment:
can be `Address` because we know that the user was successfully logged in at this step, don't we?

Reply:
My goal was to reduce the amount of domain entites in the CLI / external layer so that it can be more decoupled

Reply:
But I see that it might be useful in the future to use the Address